katharsis-core/src/main/java/io/katharsis/core/internal/utils/ClassUtils.java

```diff
@@ -92,21 +95,16 @@ public static Field findClassField(Class<?> beanClass, String fieldName) {
 	}
 
 	public static Method findGetter(Class<?> beanClass, String fieldName) {
-		String upperCaseName = fieldName.substring(0, 1).toUpperCase() + fieldName.substring(1);
-		try {
-		    return beanClass.getMethod("get" + upperCaseName);
-		} catch (NoSuchMethodException e) {
-			try {
-				Method method = beanClass.getMethod("is" + upperCaseName);
-				Class<?> returnType = method.getReturnType();
-				if(returnType == Boolean.class || returnType == boolean.class){
-					return method;
-				}
-				return null;
-			} catch (NoSuchMethodException e1) {
-				return null;
-			}
+		for (Method method : beanClass.getMethods()) {
```

Comment:
instead of 3 seperate `if/continue` arrangements. You could just have 1 `if` that recognizes the correct case. The `for` loop naturally continues.